<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Return</class>
 <widget class="QMainWindow" name="Return">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Book Returns</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLineEdit" name="bookSearch">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">outline:none;
border:0.5px solid #333;
padding:7px 15px;
font-size:15px;</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="searchBtn">
            <property name="minimumSize">
             <size>
              <width>100</width>
              <height>35</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true">padding:0 10px;
font-size:15px;</string>
            </property>
            <property name="text">
             <string>Search</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size:15px;</string>
          </property>
          <property name="text">
           <string>On Issue Books</string>
          </property>
          <property name="buddy">
           <cstring>tableView</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QTableView" name="tableView">
          <property name="minimumSize">
           <size>
            <width>800</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">font-size:15px;</string>
          </property>
          <property name="text">
           <string>Return Date</string>
          </property>
          <property name="buddy">
           <cstring>returnDate</cstring>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QDateEdit" name="returnDate">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">outline:none;
border:0.5px solid #333;
padding:7px 15px;
font-size:15px;</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="refresh">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:0 10px;
font-size:15px;</string>
          </property>
          <property name="text">
           <string>Refresh</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="confirmBtn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:0 10px;
font-size:15px;</string>
          </property>
          <property name="text">
           <string>Confirm Return</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="exitBtn">
          <property name="minimumSize">
           <size>
            <width>100</width>
            <height>35</height>
           </size>
          </property>
          <property name="styleSheet">
           <string notr="true">padding:0 10px;
font-size:15px;</string>
          </property>
          <property name="text">
           <string>Exit</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
